<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="de.uka.ilkd.key.testcase.smt.ce.TestCE" tests="12" skipped="0" failures="0" errors="0" timestamp="2023-03-24T08:26:32" hostname="fv-az587-632" time="10.052">
  <properties/>
  <testcase name="testOverFlow1()" classname="de.uka.ilkd.key.testcase.smt.ce.TestCE" time="0.088"/>
  <testcase name="testOverFlow2()" classname="de.uka.ilkd.key.testcase.smt.ce.TestCE" time="0.002"/>
  <testcase name="testMiddle()" classname="de.uka.ilkd.key.testcase.smt.ce.TestCE" time="9.941"/>
  <testcase name="testTypes1()" classname="de.uka.ilkd.key.testcase.smt.ce.TestCE" time="0.001"/>
  <testcase name="testTypes2()" classname="de.uka.ilkd.key.testcase.smt.ce.TestCE" time="0.0"/>
  <testcase name="testTypes3()" classname="de.uka.ilkd.key.testcase.smt.ce.TestCE" time="0.0"/>
  <testcase name="testTypes4()" classname="de.uka.ilkd.key.testcase.smt.ce.TestCE" time="0.0"/>
  <testcase name="testTypes5()" classname="de.uka.ilkd.key.testcase.smt.ce.TestCE" time="0.001"/>
  <testcase name="testTypes6()" classname="de.uka.ilkd.key.testcase.smt.ce.TestCE" time="0.0"/>
  <testcase name="testTypes7()" classname="de.uka.ilkd.key.testcase.smt.ce.TestCE" time="0.0"/>
  <testcase name="testTypes8()" classname="de.uka.ilkd.key.testcase.smt.ce.TestCE" time="0.001"/>
  <testcase name="testTypes9()" classname="de.uka.ilkd.key.testcase.smt.ce.TestCE" time="0.0"/>
  <system-out><![CDATA[08:26:32,198 |-INFO in ch.qos.logback.classic.LoggerContext[default] - This is logback-classic version 1.4.6
08:26:32,214 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
08:26:32,223 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback.xml] at [file:/home/runner/work/key/key/key.core.testgen/build/resources/test/logback.xml]
08:26:32,304 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - Processing appender named [STDOUT]
08:26:32,304 |-INFO in ch.qos.logback.core.model.processor.AppenderModelHandler - About to instantiate appender of type [ch.qos.logback.core.ConsoleAppender]
08:26:32,327 |-WARN in ch.qos.logback.core.model.processor.ImplicitModelHandler - Ignoring unknown property [file] in [ch.qos.logback.core.ConsoleAppender]
08:26:32,327 |-WARN in ch.qos.logback.core.model.processor.ImplicitModelHandler - Ignoring unknown property [append] in [ch.qos.logback.core.ConsoleAppender]
08:26:32,327 |-INFO in ch.qos.logback.core.model.processor.ImplicitModelHandler - Assuming default type [ch.qos.logback.classic.encoder.PatternLayoutEncoder] for [encoder] property
08:26:32,340 |-INFO in ch.qos.logback.classic.model.processor.RootLoggerModelHandler - Setting level of ROOT logger to WARN
08:26:32,344 |-INFO in ch.qos.logback.core.model.processor.AppenderRefModelHandler - Attaching appender named [STDOUT] to Logger[ROOT]
08:26:32,344 |-INFO in ch.qos.logback.core.model.processor.DefaultProcessor@77307458 - End of configuration.
08:26:32,345 |-INFO in ch.qos.logback.classic.joran.JoranConfigurator@1fc0053e - Registering current configuration as safe fallback point

317        WARN  Test worker     d.u.i.k.t.s.c.TestCE      Warning: Z3_CE is not installed, tests skipped. 
319        WARN  Test worker     d.u.i.k.t.s.c.TestCE      Maybe use JVM system property "z3SolverPath" to define the path to the Z3 command. 
364        WARN  Test worker     d.u.i.k.s.ProofIndependentSettings The settings in /home/runner/.key/proofIndependentSettings.props are *not* read due to flag 'key.disregardSettings' 
635        WARN  Test worker     d.u.i.k.s.ProofSettings   The settings in /home/runner/.key/proof-settings.props are *not* read. 
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
